uv.lock

```diff
@@ -179,7 +179,7 @@ wheels = [
 
 [[package]]
 name = "aviary-gsm8k"
-version = "0.9.2.dev4+gf68800e.d20241104"
+version = "0.9.2.dev3+gb98e720"
```

Comment:
Let's leave the `uv lock` changes out of this PR, unless they were necessary somehow?

Reply:
Okay sounds good, sure thing. Can you rebase atop `main` for the CI fix? Also, can you just revert the `uv.lock` changes in this PR? We don't need them